src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -65,10 +73,13 @@ export class CalciteCombobox {
   /** Placeholder text for input */
   @Prop() placeholder?: string;
 
-  /** specify the maximum number of combobox items (including nested children) to display before showing the scroller */
+  /** Specify the maximum number of combobox items (including nested children) to display before showing the scroller */
   @Prop() maxItems = 0;
 
-  /** specify the scale of the combobox, defaults to m */
+  /** Allow entry of custom values which are not in the original set of items */
+  @Prop() customValues: boolean;
```

Comment:
Maybe something like `allowEntry` or `allowAdd`?

Reply:
Yeah I like something like “allow-entry” or “allow-custom-values” here

Reply:
I do like how explicit the `allow` prefix is. Not sure about entry or add as that is kind of vague. 

I'm going to update it to use `allow-custom-values`. It's long but very clear what it does.